packages/docs/playwright/gnosisSafeUtils.ts

```diff
@@ -1,22 +1,30 @@
 import { BrowserContext, Page } from 'playwright'
 import waitForExpect from 'wait-for-expect'
 import { expect } from 'chai'
-import { waitForPopup } from '@usedapp/playwright'
+import { waitForPopup, XPath } from '@usedapp/playwright'
 import debug from 'debug'
+import { sleep } from './sleep'
 
-export const GNOSIS_SAFE_URL = 'https://gnosis-safe.io/app/gor:0xA971C98755c3404Fc4458fcd98905980f68Af642/home'
+export const GNOSIS_SAFE_URL = 'https://app.safe.global/gor:0xA971C98755c3404Fc4458fcd98905980f68Af642/home'
 
 const log = debug('usedapp:docs:playwright')
 
 export async function initGnosisSafe({ page, url }: { page: Page; url: string }) {
   await page.goto(url)
-  await page.click('//span[contains(text(), "Accept all")]')
-  await page.click('//p[contains(text(), "WalletConnect")]')
+  await page.click(XPath.text('button', 'Accept all'))
+  try {
+    await page.click('//*[local-name()="svg" and @data-testid="CloseIcon"]', { timeout: 2000 })
```

Comment:
Add a comment saying what we're trying to close here. Assuming a cookie popup or something not relevant to the flow, right?